src/neo/Trie/MPT/MPTNode.cs

```diff
@@ -0,0 +1,102 @@
+using Neo.Cryptography;
+using Neo.IO;
+using System.IO;
+using System.Text;
+
+namespace Neo.Trie.MPT
+{
+    public enum NodeType
+    {
+        FullNode,
+        ShortNode,
+        HashNode,
+        ValueNode,
+    }
```

Comment:
For me, I'd like to assign them clear values to ensure consistency in other languages.
```c#
public enum NodeType : byte
    {
        FullNode = 0x00,
        ShortNode = 0x01,
        HashNode = 0x02,
        ValueNode = 0x03,
    }
```